src/Http/Transport/CurlTransport.php

```diff
@@ -186,42 +186,45 @@ private static function mergeCurlOptions(array ...$options): array
      *
-     * @param  \CurlHandle|resource $curlHandle
+     * @param \CurlHandle|resource $curlHandle
+     *
      * @return array{0:             string, 1: string[]} First key headers, second key is content
+     *
      * @throws NetworkException
-     * @link https://stackoverflow.com/questions/10589889/returning-header-as-array-using-curl
+     *
+     * @see https://stackoverflow.com/questions/10589889/returning-header-as-array-using-curl
      */
     private static function getHeadersAndContentFromCurlHandle($curlHandle): array
     {
-        /** @noinspection PhpElementIsNotAvailableInCurrentPhpVersionInspection */
+        /* @noinspection PhpElementIsNotAvailableInCurrentPhpVersionInspection */
         // @codeCoverageIgnoreStart
         if (\PHP_VERSION_ID >= 80000 && !$curlHandle instanceof \CurlHandle) {
-            /** @noinspection PhpElementIsNotAvailableInCurrentPhpVersionInspection */
+            /* @noinspection PhpElementIsNotAvailableInCurrentPhpVersionInspection */
             throw new \InvalidArgumentException(sprintf('$curlHandle must be "%s". "%s" given.', \CurlHandle::class, get_debug_type($curlHandle)));
-        } elseif (\PHP_VERSION_ID < 80000 && !is_resource($curlHandle)) {
-            throw new \InvalidArgumentException(sprintf('$curlHandle must be resource. "%s" given.', is_object($curlHandle) ? get_class($curlHandle) : gettype($curlHandle)));
+        } elseif (\PHP_VERSION_ID < 80000 && !\is_resource($curlHandle)) {
+            throw new \InvalidArgumentException(sprintf('$curlHandle must be resource. "%s" given.', \is_object($curlHandle) ? \get_class($curlHandle) : \gettype($curlHandle)));
         }
         // @codeCoverageIgnoreEnd
 
         $headers = [];
 
-        curl_setopt($curlHandle, CURLOPT_HEADER, 1);
+        curl_setopt($curlHandle, \CURLOPT_HEADER, 1);
         $responseContent = curl_exec($curlHandle);
 
         // @codeCoverageIgnoreStart
         if (false === $responseContent) {
-            throw new NetworkException('Error during curl_exec. Error: ' . curl_error($curlHandle));
+            throw new NetworkException('Error during curl_exec. Error: '.curl_error($curlHandle));
         }
         // @codeCoverageIgnoreEnd
 
-        $headerSize    = curl_getinfo($curlHandle, CURLINFO_HEADER_SIZE);
+        $headerSize = curl_getinfo($curlHandle, \CURLINFO_HEADER_SIZE);
         $headerContent = substr($responseContent, 0, $headerSize);
 
         // Split the string on every "double" new line.
         $headerParts = explode("\r\n\r\n", $headerContent, 2); // only split once to mitigate scrapping content if it contains newlines with carriage return
 
         // Loop of response headers. The "count() -1" is to avoid an empty row for the extra line break before the body of the response.
-        for ($index = 0; $index < count($headerParts) - 1; $index++) {
+        for ($index = 0; $index < \count($headerParts) - 1; ++$index) {
```

Comment:
What do you think about the increment style?

Reply:
I would prefer the old one. But: It partly depends on the situation.

E.g.

~~~php
$i = 0;
$j = $i++;
~~~

$j is now 0.

~~~php
$i = 0;
$j = ++$i;
~~~

$j is now 1.

Personally, I would stick on the old style (more spread one) and enforce nothing regarding this point via CS as it is more than CS and can imply logical changes in some situations.

Reply:
That was my concern as well, I added a rule to prevent this.